OpenIPSL/Electrical/Controls/PSSE/TG/HYGOV.mo

```diff
@@ -14,6 +14,7 @@ model HYGOV "HYGOV - Hydro Turbine-Governor model"
   parameter Types.PerUnit A_t=1.2 "Turbine gain";
   parameter Types.PerUnit D_turb=0.2 "Turbine damping";
   parameter Types.PerUnit q_NL=0.08 "Water flow at no load";
+  parameter Types.PerUnit h0=1.0 "Water head initial value > 0.0, default 1.0";
```

Comment:
```suggestion
  parameter Types.PerUnit h0(min=0)=1 "Water head initial value > 0";
```
The default is shown in the tool anyway so no need to mention it. And if the head needs to be larger than 0 then you might just as well use the Modelica mechanism to ensure this.

Reply:
Good idea!

Reply:
Yes since tools don't display the "min" value but only warn once it is violated.

Reply:
Ok, got it!